packages/server/src/Interface.DocumentStore.ts

```diff
@@ -1,22 +1,94 @@
-import { DocumentStoreStatus } from '../../Interface'
-import { DocumentStore } from '../../database/entities/DocumentStore'
+import { DocumentStore } from './database/entities/DocumentStore'
```

Comment:
just so we are consistent, are we separating the interfaces like this for each entity going forward? Is this the best way?

Reply:
With other large modules on the way, a single Interface.ts was becoming too large and unmanageable.